src/screens/SignUpScreen.tsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import ThirdPartyButton from '../components/ThirdPartyButton';
 
 const SignUpScreen = () => {
-  return <div>SignUpScreen</div>;
+  return (
+    <ThirdPartyButton
+      imgSrc="./assets/google-icon.svg"
+      handleClick={() => {}}
```

Comment:
Add a TODO comment for the work left pending 👍🏽 (creating a ticket and adding it to the comment would also be great that way we know what is left to do and we can scope it out during a meeting)